### PR TITLE
fix:style fixed

### PR DESCRIPTION
**Pull request by florashore**
fixed style

### CODE REVIEW

**Review by coderprovider — COMMENTED** [review]
It is good

File: src/components/datePicker/datePicker2.js [hunk 1]
```diff
@@ -46,13 +46,11 @@ const DatePicker2 = (
 
   return (
     <div className={` ${datePickerStyle} absolute z-10 p-[15px] mt-[-30px] w-[29%] overflow-hidden bg-zinc-900 border-[1px] border-[#FFFFFF33] text-white rounded-[10px]`}>
-
```
Comment (Jermaine522) [inline src/components/datePicker/datePicker2.js:49]: 😀

**Review by Jermaine522 — COMMENTED** [review]
excellent

**Review by miraclaers — COMMENTED** [review]
👍

**Comment by florashore** [comment]
totally good

**Pull request merged**